File: tokilearning/Latihan/ratarata.pas
var
  n,i:longint;
  max,min,avg,a:extended;

begin
  avg:=0; min:=1e50; max:=-1e50;
  readln(n);
  for i:=1 to n do
    begin
      readln(a);
      avg:=avg+a/n;
      if a>max then max:=a;
      if a<min then min:=a;
    end;
  writeln(min:0:2,' ',max:0:2,' ',avg:0:2);
end.

File: tokilearning/Latihan/ratarata.cpp
#include <stdio.h>

using namespace std;

int main(){
    int n;
    double max,min,sum,z;
    
    scanf("%d %lf", &n, &sum);
    max=min=sum;
    
    for (int i=1; i<n; i++){
        scanf("%lf",&z);
        if (z>max) max=z;
        if (z<min) min=z;
        sum+=z;
        }
    sum/=n;
    
    printf("%.2f %.2f %.2f\n",min,max,sum);
    
    return 0;
}
